### DISCUSSION TITLE
Cohort analysis

### DISCUSSION THREAD

**Discussion by metmarkosaric** [plausible/analytics#364] [Anything Else]
"Your tool already has the proper basis for cohort analysis with the filters. Meta-data linked to goals is also a good step ahead. You are definitely on the good track.

Nevertheless I think your roadmap still needs a proper work about cohort analysis.
What I’m looking for is a comparison of traffic based on filters. So you can say that it’s multi-filtering.

I would like to differentiate (and compare) the behavior of 2 cohorts (or more) based on simple and advanced criteria.

For example: what is the transformation rate for a given CTA of mobile users VS desktop users ?

This particular example would allow me to highlight the fact that a button is not well positioned for mobile users. That is exactly this kind of stats that allow me to take data-driven decisions.

For this particular example, we realized that the main CTA “contribuer” on the homepage was to move up above the fold (600px height).

Ideally the tool would allow to compare transformation rate of objectives over time. The filters that I’m thinking of are software config (browser, os, size resolution), objectives (display of a specific URL, click on a given button).

Example of an advanced cohort analysis with more than 2 criteria : comparison of the evolution from last year to this year of the transformation rate of a given CTA for android user VS ios users.

See this G.Analytics example with 3 cohorts:"

![e4ff76af83fbbd237d51256dde80b144012cdf74](https://user-images.githubusercontent.com/34340819/96098570-8f98aa00-0ed2-11eb-9d06-e63b50bccfac.png)

(feature request [from the forum](https://plausible.discourse.group/t/cohort-analysis/91/5))

**Comment by overflorian** [comment]
Hello,
Could you please provide an update regarding cohort's implementation? I've been checking the doc today and it seems that it's not implemented yet.
Thanks

**Reply by metmarkosaric** [reply]
hi @overflorian! there are no short term plans for this so we cannot make any promise on when it will be done. we are a small team with limited development resources so need to focus on one thing at a time with priority being on the more frequently requested features. thanks for understanding!

**Reply by overflorian** [reply]
Hi, of course I definitely understand your constraints.
I feel really sorry not to be able to use your great software for now, due to this limitation. Please keep me updated about this topic of possible.
I remain really interested in your project, take care

**Reply by metmarkosaric** [reply]
thank you! we will update this thread as soon as there's any progress on this
